test/src/test/resources/proto/sample_schema.proto

```diff
@@ -78,6 +78,8 @@ message SampleTableBMsg {
     option (org.corfudb.runtime.table_schema).stream_tag = "sample_streamer_3";
     option (org.corfudb.runtime.table_schema).requires_backup_support = false;
     option (org.corfudb.runtime.table_schema).is_federated = true;
+    option (org.corfudb.runtime.table_schema).logical_group = "group1";
+    option (org.corfudb.runtime.table_schema).client_name = "logical_group_consumer";
```

Comment:
In my opinion, if 'logical group' option field is present but not 'client name' it would be an user error and likely this table will not be replicated as it won't be associated to any client. To prevent this, could we instead define it in such a way that both fields are forced to be defined, like a complex type?

Reply:
That is a very good suggestion - will do as you say! thank you